packages/utils/src/types.ts

```diff
@@ -31,6 +31,10 @@ export type RJSFSchema = StrictRJSFSchema & GenericObjectType;
  */
 export type FormContextType = GenericObjectType;
 
+export type Experimental_DefaultFormStateBehavior = {
+  arrayMinItems?: 'populate' | 'requiredOnly';
+};
```

Comment:
Would like this types documentation to match what's in the docs project, not a big deal while this is experimental though

Reply:
I just mean having the supplemental text here as well so IDEs can provide the documentation info at a glance, so one does not have to open the docs page to know what each option does